### PR TITLE
Ability to resolve environment variables in JSON steps

### PR DESCRIPTION
**Pull request by santhoshTpixler**
Implements #282 Ability to resolve environment variables in JSON steps.

### CODE REVIEW

File: core/src/main/java/org/jsmart/zerocode/core/utils/TokenUtils.java [hunk 1]
```diff
@@ -134,4 +139,9 @@ public static String getXmlContent(String xmlFileResource) {
                     + "', details:" + e);
         }
     }
+
+    public static void main(String[] args) {
```
Comment (authorjapps) [inline core/src/main/java/org/jsmart/zerocode/core/utils/TokenUtils.java]: Can you please remove this main()?

File: core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java [hunk 2]
```diff
@@ -98,4 +99,12 @@ public void testFixedRandomUniqueness() {
         assertTrue(split[0] != split[1]);
     }
 
+
+    @Test
+    public  void testEnvPropertyReplace(){
+        String text = "${SYSTEM.ENV:PWD}";
```
Comment (authorjapps) [inline core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java]: May be an assertion with a know env value of `JAVA_HOME` types will help

File: core/src/main/java/org/jsmart/zerocode/core/utils/TokenUtils.java (hunk 1 above)
Comment (santhoshTpixler) [reply core/src/main/java/org/jsmart/zerocode/core/utils/TokenUtils.java]: Sorry forget forgot to remove

File: core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java (hunk 2 above)
Comment (santhoshTpixler) [reply core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java]: Yeah, `JAVA_HOME` can be used, but how can we verify it?
In the case of `PWD`, we can verify it by fetching the current directory from the Paths API

**Comment by authorjapps** [comment]
And an example in `http-testing` module please 👍

File: core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java (hunk 2 above)
Comment (authorjapps) [reply core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java]: right, that's ok. Understood.
Then just add
`assertThat(....)` bit to the test 👍

Comment (santhoshTpixler) [reply core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java]: Yeah, Forgot the assertion part :man_facepalming:

Comment (santhoshTpixler) [reply core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java]: I have modified the test to make more sense. I guess it should be okay. Feel free to throw your suggestions.

**Comment by santhoshTpixler** [comment]
Kindly squash the commits while merging.

**Pull request merged**